test/jasmine-test/spec/coreTests.js

```diff
@@ -724,8 +748,43 @@ var coreTests = [
         context:  { foo: {bar: "Hello!"} },
         expected: "Hello!",
         message: "should test an object path"
-      }
-    ]
+      },
+	  {
+		name:     "thenable reference",
+		source:   "Eventually {magic}!",
```

Comment:
A Thenable is not a function so it would look for a property on the Object called `foo`